atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -1642,9 +1642,18 @@ private void putMetadataForTablesInternal(
     }
 
     private void putMetadataWithoutChangingSettings(final TableReference tableRef, final byte[] meta) {
+        long ts = System.currentTimeMillis();
```

Comment:
It's not ideal, it's just a hack already in use in our past impls of this.

In general, I'd like to remove metadata from the KVS layer entirely except for on createTable (along with the additional guarantee listed in the comment above that would ensure updates to KVS-layer settings could occur on existing table types) and do table metadata magic in a higher atlas layer, much akin to the special tables interacted with by the Puncher or SweepStats et al that sort of make up a loosely-defined core system schema for atlas.